doc/welcome.rst

```diff
@@ -0,0 +1,21 @@
+Welcome
+=======
+
+Welcome to the Cabal user guide.
+Cabal is the "common architecture for building applications and tools" and consists of a fileformat for describing Haskell packages, a library for working with those files, and a commandline tool for downloading, building and running Haskell applications and libraries.
+
```

Comment:
If the lower part becomes a dedicated page, would this section make more sense on the getting-started page?